### ISSUE TITLE
DIE fails to load on ida for linux

### ISSUE THREAD

**Issue by crowell** [ynvb/DIE#17]
import of networkx fails

![image](http://i.imgur.com/OwkdGHV.png)

**Comment by crowell** [comment]
also confirmed failure on OS X

**Comment by tmr232** [comment]
Did you install the requirements? (using `pip install -r requirements.txt` in DIE's top directory) 
It should solve it.

**Comment by crowell** [comment]
@tmr232 yes i did this, nope it didn't solve the issue

**Comment by ynvb** [comment]
I have never tested it on OSX\Linux versions of IDA.
Have yo tries installing networkx manually?

**Comment by crowell** [comment]
@ynvb yes, i installed from pip (system python) pip (virtualenv), and from the ubuntu repository.

**Comment by ltfish** [comment]
I think it's because your IDA installation is using the Python bundled with IDA itself, not the global one. if that's the case, you cannot execute `import networkx` in IDAPython - since you didn't install `networkx` for the bundled Python (with IDA).

Check out `~/ida-6.x/python` to see if that's the case. You may simply symlink your installation of `networkx` to '~/ida-6.x/python/networkx`, and see if the problem solves.

This was a big pain when we were developing `idalink` :-(

**Comment by tmr232** [comment]
This solution had two main issues:
1. It will require adding a similar symlink to each and every dependency;
2. If the system python is not compatible to IDA's version it might cause weird errors. 

If you have a compatible python version install, you can probably use one of the following solutions (untested as I do not have IDA for Linux):
1. Symlink the entire `site-packages` directory, or use a `.pth` to link it;
2. Use a virtualenv like script to "activate" IDA's python environment, then use pip normally. 

I hope this helps.

**Comment by crowell** [comment]
with virtualenv + [this](https://github.com/Kerrigan29a/idapython_virtualenv) plugin i got it working, debugging throws some errors about numbers being too large to convert to int, I'll try to dig a bit and give you a better bug report when i have time to dig into it.
